packages/testing/playwright/pages/n8nPage.ts

```diff
@@ -74,7 +72,6 @@ export class n8nPage {
 		this.demo = new DemoPage(page);
 		this.iframe = new IframePage(page);
 		this.interactions = new InteractionsPage(page);
-		this.logs = new LogsPage(page);
```

Comment:
Ideally we'd need to have a good idea of what we think should be a "page" now. Since technically NDV isn't a page, but it's a massive object that probably deserves it's own page object. Something for us to think on in the future.

Reply:
That's going to be interesting so!